eventhub/Package.md

```diff
@@ -1,540 +1,23 @@
-# Ballerina Azure Event Hubs Module
+Connects to Microsoft Azure Event Hubs from Ballerina.
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
+## Package overview
 
-# Module Overview
+The `ballerinax/azure_eventhub` is a [Ballerina](https://ballerina.io/) connector for Microsoft Azure Event Hubs.
 
-Azure Event Hubs Ballerina Connector is used to connect with the Azure Event Hubs to ingest millions of events per 
-second so that you can process and analyze the massive amounts of data produced by your connected devices and 
-applications. Once data is collected into an Event Hub, it can be transformed and stored using any real-time 
-analytics provider or batching/storage adapters.
+This package provides the capability to easily access Microsoft Azure Event Hubs to ingest millions of events per second 
```

Comment:
remove unnecessary new lines.

Reply:
Resolved and amended the changes. Thank you for the suggestion.